.github/workflows/core_build_publish.yml

```diff
@@ -152,44 +151,40 @@ jobs:
           CORELIB_TESTS_AZURE_BLOB_STORAGE_CONTAINER_NAME: ${{ secrets.CORELIB_TESTS_AZURE_BLOB_STORAGE_CONTAINER_NAME }}
           CORELIB_TESTS_AZURE_TABLE_STORAGE_TABLE_NAME: ${{ secrets.CORELIB_TESTS_AZURE_TABLE_STORAGE_TABLE_NAME }}
       - name: Integration Test - SQL Server
-        run: dotnet test -c Release --logger trx --results-directory ../TestResults
-        working-directory: test/LeanCode.IntegrationTests
+        run: dotnet test -- --explicit on --filter-query '/[(category=integration)&(database!=postgres)]'
```

Comment:
why do you filter here by `postgres` and not in the `Integration Test - Postgres`?

Reply:
We support two databases and we have more tests for Postgres than for SQL Server. Here, I filter-out Postgres-specific tests (there's inequality operator).